README.md

```diff
@@ -1,8 +1,8 @@
 # OracleMinimalApp-iOS
 Oracle Minimal App integration for iOS
-The Bluedot SDK integration with Oracle Responsys Mobile App Platform Cloud Service enables mobile apps to take advantage of the power of  Cloud services and the superior accuracy, Geofence, Geolines™ and BLE Beacon triggering capabilities of the Bluedot SDKs. 
+The Bluedot SDK integration with Oracle Responsys Mobile App Platform Cloud Service enables mobile apps to take advantage of the power of  Cloud services and the superior accuracy, Geofence, Geolines™ and BLE Beacon triggering capabilities of the Bluedot SDKs.
```

Comment:
This README is still a little unclear on the need to finish integrating the Responsys SDK, let's bring it into alignment with the Android readme:

```suggestion

The Bluedot SDK integration with Oracle Responsys Mobile App Platform Cloud Service enables mobile apps to take advantage of the power of Oracle Cloud services and the superior accuracy, Geofence, Geolines™ and BLE Beacon triggering capabilities of the Bluedot SDKs.

### To add the Oracle SDK
Please follow detailed instructions at https://docs.oracle.com/en/cloud/saas/marketing/responsys-develop-mobile/android/android.htm

### To integrate Bluedot SDK
Please refer integration steps mentioned at https://docs.bluedot.io/integrations/oracle-integration/ios/
```

Reply:
Readme updated